packages/docusaurus-theme-openapi/src/theme/ApiItem/index.tsx

```diff
@@ -10,7 +10,7 @@ import React from "react";
 import ExecutionEnvironment from "@docusaurus/ExecutionEnvironment";
 import type { Props } from "@theme/ApiItem";
 import DocPaginator from "@theme/DocPaginator";
-import Seo from "@theme/Seo";
+import { PageMetadata } from "@docusaurus/theme-common";
```

Comment:
alphabetize haha :)

Reply:
where do you want me to move it exactly? Not sure on what criteria to alphabetize :)

Reply:
Above @theme/ApiItem